README.md

```diff
@@ -71,10 +71,11 @@ gets installed into the `/usr/local/share/zsh/site-functions/` directory.
 
 The `mender-cli` tool can be enabled to support shell autocompletion, like you
 are used to for your regular tools, like `git`, `cd`, etc. In order to enable
-this functionality run the `mender-cli` tool with the `--generate` flag. Example:
+this functionality run the `mender-cli` tool with the `--generate-autocomplete`
+flag. Example:
 
 ```console
-mender-cli --generate
+mender-cli --generate-autocomplete
```

Comment:
@oleorhagen This doesn't work for me, it just does nothing. Can you check in your end?

Reply:
Thanks. I will merge regardless as it is not related to my PR. Maybe something wrong with my `bash`, for all I know.

Reply:
Nope, not working for me either now :thinking: 

_Proceeds to read Changelogs_